dhcp-controller/api/v1alpha1/dhcpserver_types.go

```diff
@@ -25,6 +25,8 @@ type DHCPServerSpec struct {
 	// Details of Networks
 	// +kubebuilder:validation:Required
 	Networks []Network `json:"networks,omitempty"`
+	// Hugepage Size
+	HugepageSize string `json:"hugepageSize,omitempty"`
```

Comment:
Ideal way would be avoid dependency on DPDK hugepages, if not this value can be derived internally , based on isDPDK flag..

Reply:
HugepageSize field is optional, so DHCPServer can be created without it.

Reply:
Fixed